### PR TITLE
Automatically send progress event when rendered with display_submit=False

### PR DESCRIPTION
**Pull request by e-kolpakov**
@bradenmacdonald please note I recreated `real-children` branch - we might still need it to port stuff from edx-solutions branches.

Also, I still got 1 error and 3 failures in my local installation, so any thought on how it works in travis and your environment are welcome (e.g. which versions of xblock-sdk and XBlock are used)

### CODE REVIEW

**Comment by antoviaque** [comment]
The code looks good, and it's the same approach as in https://github.com/edx-solutions/xblock-mentoring/pull/99 so it should work well, but I wasn't able to test the new version of the mentoring XBlock, and thus this change. I'm going to post about the issues I've had in OC-513, but in the meantime if this works for the other reviewer this can be merged, as we only need one :+1: on this repo.

File: mentoring/mentoring.py [hunk 1]
```diff
@@ -98,7 +98,12 @@ def is_default_xml_content(value):
         scope=Scope.content,
         enforce_type=True
     )
-    display_submit = Boolean(help="Allow to submit current block?", default=True, scope=Scope.content)
+    display_submit = Boolean(
+        help="Allow to submit current block?",
```
Comment (Kelketek) [inline mentoring/mentoring.py]: Same comment here as in the other PR.

**Comment by e-kolpakov** [comment]
@Kelketek done

**Comment by antoviaque** [comment]
@Kelketek Same comment here too about giving a :+1: - also where you able to test it with the real children version?

**Comment by Kelketek** [comment]
@antoviaque I was able to run the tests enough to verify that this approach should be correct, but I am also having general issues with dependencies when trying to test this thing in general. The other version of this repo, which has the self-contained testing apparatus, reliably passes.

As a result of these problems, I can't get completely passing test build, but I'm pretty sure that this particular changeset has nothing to do with that, so I'll give it a :+1: .

I take it that we're going to do some sort of rebase/merge down the line that will allow us to have both real children and that test management, yes? :)

**Comment by Kelketek** [comment]
@e-kolpakov @antoviaque On second thought, something about this bothers me and I want to investigate further, just to be sure. @e-kolpakov could you give me a pip freeze readout?

**Comment by Kelketek** [comment]
Actually, I'm probably just being paranoid. I rolled back your commit to see if the same failures occurred without it, and they do.

We're good to go, I think.

**Pull request merged**